README.md

```diff
@@ -10,6 +10,13 @@ Provides pbot commands to help us manage our issues.
 3. The bot is setup to watch the TC nuget feed so just go into a channel and type `pbot update package PBot restart`
 4. The bot will now update and restart, this take 1-2 min, just check that the bot is alive using `pbot ping`
 
+##### Troubleshooting
+
+If the package fails to update, check that:
+* the master branch has indeed been built, creating the new version
+* the build artifacts contain the NuGet package
+* the TeamCity NuGet feed is working. If empty, a TeamCity restart my be required.
```

Comment:
It turns out that checking the feed is working isn't enough. We need to figure out a way of ensuring that the new version of a package from a new build is added to the feed.

Another problem we've noticed is that, after resetting the buildsMetadata cache under Administration -> Diagnostics, the cache stays empty and therefore the NuGet feeds stays empty. This seems at odds with https://confluence.jetbrains.com/display/TCD9/Common+Problems#CommonProblems-ProblemswithTeamCityNuGetFeed. It seems that there is a more general problem with indexing.